cluster/gce/util.sh

```diff
@@ -1136,6 +1139,11 @@ EOF
   if [ -n "${SCHEDULING_ALGORITHM_PROVIDER:-}" ]; then
     cat >>$file <<EOF
 SCHEDULING_ALGORITHM_PROVIDER: $(yaml-quote ${SCHEDULING_ALGORITHM_PROVIDER})
+EOF
+  fi
+  if [ -n "${MAX_PODS:-}" ]; then
+    cat >>$file <<EOF
+MAX_PODS: $(yaml-quote ${MAX_PODS})
```

Comment:
Should this env be more specific? e.g. `MAX_PODS_PER_NODE`, `MAX_PODS_PER_KUBELET`?

Reply:
Renamed. MAX_PODS_PER_NODE picked since this env might be used somewhere else in the future, not only for kubelet.